rtc-video-quality/generate_data.py

```diff
@@ -384,7 +384,7 @@ def add_framestats(results_dict, framestats_file, statstype):
   with open(framestats_file) as csvfile:
     reader = csv.DictReader(csvfile)
     for row in reader:
-      for (metric, value) in row.items():
+      for (metric, value) in list(row.items()):
```

Comment:
Is list() necessary here? I believe items already returns a list.

Reply:
I think this is done by the script. I'll update the code 👍